experimental/dds/tree2/src/shared-tree/sharedTree.ts

```diff
@@ -238,7 +243,27 @@ export class SharedTree
 /**
  * @alpha
  */
-export interface SharedTreeOptions extends Partial<ICodecOptions> {}
+export interface SharedTreeOptions extends Partial<ICodecOptions> {
+	/**
+	 * The {@link ForestType} indicating which forest type should be created for the SharedTree.
+	 */
+	forest?: ForestType;
```

Comment:
The default is not stated here, nor is it explicit in the implementation where this is used.

I good pattern I like for this kind of things is to export const that has the defaults (ex: const defaultOptions: Required<SharedTreeOptions> = {......};`

Then when used in Shared tree you can do: `const fullOptions = {...defaultOptions, ...options};

I think that approach scales better to more options, and makes it easy to change what the defaults are in a single place.

Reply:
Added and exported `defaultSharedTreeOptions` to be used in sharedTree